spec/features/sign_in_spec.rb

```diff
@@ -45,63 +45,142 @@
     expect(root_page).to be_displayed
   end
 
-  scenario "new user accepts the transition info page" do
-    allow(Settings).to receive(:rollover).and_return(true)
-    user = build :user, :new
-
-    stub_omniauth(user: user)
-    request = stub_api_v2_request "/users/#{user.id}/accept_transition_screen", user.to_jsonapi, :patch
-
-    visit "/signin"
-
-    expect(transition_info_page).to be_displayed
-
-    expect(transition_info_page.title).to have_content("Important new features")
-    transition_info_page.continue.click
-
-    expect(rollover_page).to be_displayed
-    expect(request).to have_been_made
-  end
-
-  scenario "new user accepts the transition info page with rollover disabled" do
-    allow(Settings).to receive(:rollover).and_return(false)
-    user = build :user, :new
-
-    stub_omniauth(user: user)
-    request = stub_api_v2_request "/users/#{user.id}/accept_transition_screen", user.to_jsonapi, :patch
-
-    visit "/signin"
-
-    expect(transition_info_page).to be_displayed
-
-    expect(transition_info_page.title).to have_content("Important new features")
-    transition_info_page.continue.click
-
-    expect(root_page).to be_displayed
-    expect(request).to have_been_made
-  end
-
-  scenario "new user accepts the rollover page" do
-    allow(Settings).to receive(:rollover).and_return(true)
-    user = build :user, :transitioned
-
-    stub_omniauth(user: user)
-    request = stub_api_v2_request "/users/#{user.id}/accept_rollover_screen", user.to_jsonapi, :patch
-
-    visit "/signin"
-
-    expect(rollover_page).to be_displayed
-
-    expect(rollover_page.title).to have_content("Begin preparing for the next cycle")
-    rollover_page.continue.click
-
-    expect(root_page).to be_displayed
-    expect(request).to have_been_made
+  describe "Interruption screens" do
+    before do
+      stub_omniauth(user: user)
+    end
+
+    context "transition screen" do
```

Comment:
```suggestion
    describe "transition screen" do
```

Reply:
Actually, if we want this to be a context, we could say:

```suggestion
    context "new user" do
```